src/iotjs_reqwrap.h

```diff
@@ -22,59 +22,38 @@
 #include "iotjs_binding.h"
 
 
-namespace iotjs {
+#ifdef __cplusplus
+extern "C" {
+#endif
 
 
 // UV request wrapper.
 // Wrapping UV request and javascript callback.
 // When an instance of request wrapper is created. it will increase ref count
 // for javascript callback function to prevent it from reclaimed by GC. The
 // reference count will decrease back when wrapper is being freed.
-template<typename T>
-class ReqWrap {
- public:
-  ReqWrap(const iotjs_jval_t* jcallback);
-  virtual ~ReqWrap();
+typedef struct {
+  iotjs_jval_t jcallback;
+  uv_req_t* request;
+} IOTJS_VALIDATED_STRUCT(iotjs_reqwrap_t);
 
-  // To retrieve javascript callback funciton object.
-  const iotjs_jval_t* jcallback();
 
-  // To retrieve pointer to uv request.
-  T* req();
+void iotjs_reqwrap_initialize(iotjs_reqwrap_t* reqwrap,
+                              const iotjs_jval_t* jcallback,
+                              uv_req_t* request,
+                              void* wrap);
```

Comment:
I think this line should be same line with ``uv_req_t* request, ``.

According to our coding style guideline - ``Return type should be on the same line as function name and parameters if it fits. If not, break between them aligned with the first argument.``

I think this rule reference to google's style guideline - https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions

It shows some example, like below:

```c
ReturnType ClassName::ReallyLongFunctionName(Type par_name1, Type par_name2,
                                             Type par_name3) {
  DoSomething();
  ...
}
```